src/wp-includes/block-template-utils.php

```diff
@@ -516,6 +516,7 @@ function _build_block_template_result_from_post( $post ) {
 	$template->status         = $post->post_status;
 	$template->has_theme_file = $has_theme_file;
 	$template->is_custom      = true;
+	$template->post_author    = $post->post_author;
```

Comment:
Should we set `$template->post_author` to something in `_build_block_template_result_from_file`? as well Or is 0 correct in that case?

Reply:
It's a good question. File templates won't have an author, but I wasn't sure what a good default value was here. I leaned on convention by using the same value posts do.

But maybe it'd be better to return no author property in the response for file templates, which could also be reflected in the WP_Block_Template class by not giving it a default of `0`.

Reply:
I think `0` as no author makes sense for file templates.